.github/docker-images/ubi8/Dockerfile

```diff
@@ -36,7 +37,7 @@ RUN curl -sSL https://github.com/Kitware/CMake/releases/download/v3.10.0/cmake-3
     && tar -zxvf cmake-3.10.0.tar.gz \
     && cd cmake-3.10.0 \
     && ./bootstrap \
-    && make -j 4 \
+    && make \
```

Comment:
QQ: why only in this docker file this change is made?

Reply:
It actually doesn't make sense to explicitly set parallelism here since the amount of parallelism depends on how many cores allocated to docker build.